packages/common/src/store/account/sagas.ts

```diff
@@ -222,10 +224,23 @@ export function* fetchAccountAsync({
     return
   }
 
-  const accountData = (yield* call([queryClient, queryClient.fetchQuery], {
-    queryKey: getWalletAccountQueryKey(wallet),
-    queryFn: async () => getWalletAccountQueryFn(wallet!, sdk)
-  })) as AccountUserMetadata | undefined
+  let accountData: AccountUserMetadata | undefined
+  try {
+    accountData = (yield* call([queryClient, queryClient.fetchQuery], {
+      queryKey: getWalletAccountQueryKey(wallet),
+      queryFn: async () => getWalletAccountQueryFn(wallet!, sdk),
+      staleTime: Infinity,
+      gcTime: Infinity
+    })) as AccountUserMetadata | undefined
+  } catch (e) {
```

Comment:
q: Should we be checking for 404 here? are there other failure modes we might mask?